app/views/layouts/application.html.erb

```diff
@@ -55,11 +55,22 @@
             </span>
           </a>
         </div>
-          <div class="govuk-header__content">
-
+        <div class="govuk-header__content">
           <a href="#" class="govuk-header__link govuk-header__link--service-name">
             Support teacher training publishers
           </a>
+
+          <button role="button" class="govuk-header__menu-button js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
+          <nav>
+            <ul id="navigation" class="govuk-header__navigation" aria-label="Top Level Navigation">
+              <li class="govuk-header__navigation-item <%= current_page?(controller: 'access_requests', action: 'index') ? 'govuk-header__navigation-item--active' : '' %>">
```

Comment:
I think it would be cleaner to pass the route in here rather than the controller and action name...

`class="govuk-header__navigation-item <%= 'govuk-header__navigation-item--active' if  current_page?(access_requests_path) %>"`

Reply:
Good call - much nicer! Fixed and force-pushed...